docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -119,3 +119,101 @@ function hs_paragraph_types_update_10001(&$sandbox) {
 
   $sandbox['#finished'] = count($sandbox['ids']) ? 1 - count($sandbox['ids']) / $sandbox['total'] : 1;
 }
+
+/**
+ * Set default field_title_settings value for existing content.
+ */
+function hs_paragraph_types_update_10002(&$sandbox) {
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  if (empty($sandbox['ids'])) {
+    // Import new field_title_settings configuration.
+    $config_storage = \Drupal::service('config.storage.sync');
```

Comment:
FWIW, I would've probably split the field creation into a separate update vs updating the field. like `10002` creates the field, `10003` populates the field. Very similar to your `10003`/`10004` updates below.